apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.Tests/SpannerDbTypeTests.ValueConversions.cs

```diff
@@ -226,15 +226,15 @@ public static IEnumerable<object[]> GetValidValueConversions()
             yield return new object[] {(short) 1, SpannerDbType.String, Quote("1")};
             yield return new object[] {(ushort) 1, SpannerDbType.String, Quote("1")};
             yield return new object[] {s_testDate, SpannerDbType.String, Quote("2017-01-31T03:15:30.5Z")};
-            //Note the difference in C# conversions from special doubles.
+            // Note the difference in C# conversions from special doubles.
             yield return new object[] {double.NegativeInfinity, SpannerDbType.String, Quote("-Infinity") };
             yield return new object[] {double.PositiveInfinity, SpannerDbType.String, Quote("Infinity") };
             yield return new object[] {double.NaN, SpannerDbType.String, Quote("NaN")};
             yield return new object[] {"1.5", SpannerDbType.String, Quote("1.5")};
             yield return new object[]
                 {new ToStringClass("hello"), SpannerDbType.String, Quote("hello"), TestType.ClrToValue};
 
-            //Spanner type = Date+Timestamp tests.  Some of these are one way due to either a lossy conversion (date loses time)
+            // Spanner type = Date+Timestamp tests.  Some of these are one way due to either a lossy conversion (date loses time)
```

Comment:
nit: "due to either a ..." or what? The either doesn't have a "or B"

Reply:
It's in the next line: "or a string formatting difference."

Reply:
duh